### ISSUE TITLE
[Bug] NVDA Screen reader doesn't read out Rich Text Field labels consistently. 

### ISSUE THREAD

**Issue by mirano-darren** [CMSgov/eAPD#4283]
### Description
With the addition of labels for the rich text fields in #2675. The NVDA screen reader does not read these labels out consistently. This is not a problem with other screen readers like Windows Narrator or VoiceOver

### Steps to reproduce
(Once 2675 gets merged in)
1. Focus on Rich Text Fields with NVDA active 
Note: This is inconsistent so it might take a little playing around with to get it to not read the label.

**Expected results-**
Label for the rich text field gets read out loud.

**Actual results-**
What actually gets read out: "Application rich text area frame, document, blank"

### Role Affected

- [ ] All
- [ ] Federal Admin
- [x] State Admin
- [x] State Staff
- [x] State Contractor
- [ ] System Admin

### Environment

- [ ] All
- [x] PR #2675
- [ ] Staging
- [ ] Production

### Browser

- [ ] All
- [ ] Firefox
- [x] Chrome
- [ ] IE

**Logs or error output/Screenshot/Video-** (at least 1 of these)
Cannot screenshot an audio bug :)
